### DISCUSSION TITLE
Possible tmux popup layout tweaks

### DISCUSSION THREAD

**Discussion by dpletiko** [jrmoulton/tmux-sessionizer#55] [Ideas]
Hi, I was wondering if the preview popup could be made to handle the vertical/horizontal monitor placement.

E.g. when using "tms switch" popup on vertical monitor, could the preview layuout switch between "rows" and "columns" depending on the monitor orientation so the preview could be wholly visible (would fill up the remaining height of the popup by moving the preview below/above list of sessions)?

And when on horizontal monitor at least use the whole height of the popup?

Thanks,
Best regards.

**Comment by jrmoulton** [comment]
It could be possible to change the orientation of the popup. We could probably just query tmux for the height/width of the tmux session and configure it if it was taller than it was wide.
